src/StepFunctionAuthoring/StepFunctionAuthoring.tsx

```diff
@@ -81,6 +85,11 @@ const StepFunctionAuthoring: FC<StepFunctionAuthoringProps> = ({ initialGraphSta
         // Detect any stored changes
         const [{ change }] = snapshots;
 
+        // Reset state as needed
+        if (change?.type === StephFunctionAuthoringChangeType.RESET) {
```

Comment:
Is there a reason this enum is spelled `StephFunction` instead of `StepFunction`?  Obviously not part of this PR, but if it's just a misspelling we should probably fix it.

Reply:
Oh good catch @adorack-tri ! I honestly didn't even realize it (using autocomplete). Will fix.